### ISSUE TITLE
Crash due to "Click" as part of Platformio

### ISSUE THREAD

**Issue by kkevlar** [mum4k/platformio_rules#22]
I've been running into a serious bug when attempting to build arduino code with this extension. PlatformIO has a Runtime error: `Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment`.

The authors suggest that this problem may be fixed by adding environment variables. They suggest:

```
This system supports the C.UTF-8 locale which is recommended. You might be able to resolve your issue by exporting the following environment variables:

    export LC_ALL=C.UTF-8
    export LANG=C.UTF-8
```

I noticed that when I run `platformio run -s -d bazel-out/k8-fastbuild/bin/robot/arduino` (the failing command) directly in my shell, the program works as expected, and there is no `Click` crash.

Full crash report:

```
/home/kevlar/Documents/roborodentia2020/robot/arduino/BUILD:15:1: error executing shell command: '/bin/bash -c unzip -qq -o -d bazel-out/k8-fastbuild/bin/robot/arduino bazel-out/k8-fastbuild/bin/robot/arduino/switch.zip
platformio run -s -d bazel-out/k8-fastbuild/bin/robot/arduino' failed (Exit 1)
Error: Traceback (most recent call last):
  File "/home/kevlar/.local/lib/python3.6/site-packages/platformio/__main__.py", line 102, in main
    cli()  # pylint: disable=no-value-for-parameter
  File "/home/kevlar/.local/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/kevlar/.local/lib/python3.6/site-packages/click/core.py", line 760, in main
    _verify_python3_env()
  File "/home/kevlar/.local/lib/python3.6/site-packages/click/_unicodefun.py", line 130, in _verify_python3_env
    " mitigation steps.{}".format(extra)
RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment. Consult https://click.palletsprojects.com/python3/ for mitigation steps.

This system supports the C.UTF-8 locale which is recommended. You might be able to resolve your issue by exporting the following environment variables:

    export LC_ALL=C.UTF-8
    export LANG=C.UTF-8

============================================================

An unexpected error occurred. Further steps:

* Verify that you have the latest version of PlatformIO using
  `pip install -U platformio` command

* Try to find answer in FAQ Troubleshooting section
  https://docs.platformio.org/page/faq.html

* Report this problem to the developers
  https://github.com/platformio/platformio-core/issues

============================================================
```

**Comment by kkevlar** [comment]
Adding the environment variables as suggested in the error message seems to fix this problem for me.

#22

**Comment by kkevlar** [comment]
Likely fixed by #23
